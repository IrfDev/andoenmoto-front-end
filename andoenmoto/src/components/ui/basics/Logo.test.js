import { mount } from '@vue/test-utils';
import Logo from './Logo.vue';

describe('Logo test snapshot', () => {
  it('Logo are the same', () => {
    const cmp = mount(Logo);
    expect(cmp.element).toMatchSnapshot();
  });
});