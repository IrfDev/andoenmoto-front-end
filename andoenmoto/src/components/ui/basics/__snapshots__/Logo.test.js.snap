// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo test snapshot Logo are the same 1`] = `
<div
  class="d-inline-block align-center"
>
  <img
    class="img-fluid d-block text-center"
    src="/identity/stars.png"
  />
   
  <img
    class="img-fluid d-block text-center"
    src="/identity/logo.png"
  />
</div>
`;

exports[`Logo test snapshot Logo are the same as always 1`] = `
<div
  class="d-inline-block align-center"
>
  <img
    class="img-fluid d-block text-center"
    src="/identity/stars.png"
  />
   
  <img
    class="img-fluid d-block text-center"
    src="/identity/logo.png"
  />
</div>
`;
